tests/zenko_tests/python_tests/zenko_e2e/conf.py

```diff
@@ -113,9 +113,9 @@ def baseurl(url):
 AZURE_CRR_TARGET_BUCKET = get_env(
     'AZURE_CRR_BUCKET_NAME', 'zenko-azure-crr-target-bucket')
 WASABI_CRR_TARGET_BUCKET = get_env(
-    'WASABI_TARGET_BUCKET', 'zenko-wasabi-crr-target-bucket')
+    'WASABI_CRR_BUCKET_NAME', 'zenko-wasabi-crr-target-bucket')
 DO_CRR_TARGET_BUCKET = get_env(
-    'DO_TARGET_BUCKET', 'zenko-digitalocean-crr-target-bucket')
+    'DO_CRR_BUCKET_BUCKET_NAME', 'zenko-digitalocean-crr-target-bucket')
```

Comment:
`DO_CRR_BUCKET_NAME`

Reply:
I just wanted to make sure we have enough `BUCKET`

Reply:
https://imgflip.com/i/2ha0mu